frontend/app/services/subscription-service.server.ts

```diff
@@ -29,19 +29,47 @@ function createSubscriptionService() {
     const auditService = getAuditService();
     const instrumentationService = getInstrumentationService();
     auditService.audit('alert-subscription.get', { userId });
-    //TODO, for a future PR, use HATEOAS links to get the /subscriptions endpoint
-    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}/subscriptions`);
-
+    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}`);
     const response = await fetch(url, {
       method: 'GET',
       headers: {
         'Content-Type': 'application/json',
       },
     });
 
+    const userSchema = z.object({
+      id: z.string(),
+      email: z.string(),
+      emailVerified: z.boolean(),
+      userAttributes: z.array(
+        z.object({
+          name: z.string(),
+          value: z.string(),
+        }),
+      ),
+      _links: z.object({
+        self: z.object({
+          href: z.string(),
+        }),
+        subscriptions: z.object({
+          href: z.string(),
+        }),
+      }),
+    });
+    instrumentationService.countHttpStatus('http.client.cdcp-api.user.gets', response.status);
+    const userParsed = userSchema.parse(await response.json());
+
+    const userSubscriptionsURL = userParsed._links.subscriptions.href;
+    const subscriptionsResponse = await fetch(userSubscriptionsURL, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
     instrumentationService.countHttpStatus('http.client.cdcp-api.alert-subscription.gets', response.status);
 
-    const newSchema = z.object({
+    const subscriptionSchema = z.object({
```

Comment:
**Nitpick:** Since this schema is for parsing more than one subscription:

```suggestion
    const subscriptionsSchema = z.object({
```

Reply:
done